src/NuGet.Clients/NuGet.CommandLine/MsBuildUtility.cs

```diff
@@ -29,7 +32,12 @@ public static bool IsMsBuildBasedProject(string projectFullPath)
             return _msbuildExtensions.Contains(Path.GetExtension(projectFullPath));
         }
 
-        public static IEnumerable<string> GetProjectReferences(string msbuildDirectory, string projectFullPath)
+        /// <summary>
+        /// Returns the closure of project references for projects specified in <paramref name="projectPaths"/>.
+        /// </summary>
+        public static Dictionary<string, HashSet<string>> GetProjectReferences(
+            string msbuildDirectory,
+            string[] projectPaths)
```

Comment:
It would be much nice if you can add some unit tests for this method, e.g. to verify that  NuGet/Home#1690 is fixed.